fusor-ember-cli/app/styles/openstack.scss

```diff
@@ -288,6 +286,10 @@
   padding: 0;
   display: block;
 }
+.deployment-roles-unassigned .draggable-object {
+  width: 150px;
+  display: inline-block;
+}
```

Comment:
@isratrade Yep, I deliberately separated out the commits to distinguish. The core fix is to use something other than a float based layout since it causes draggable objects to break in chrome.